scripts_common.py

```diff
@@ -76,6 +76,24 @@ def git_status(file):
         return None
 
 
+def username(file):
+    """
+    Return a string with the username of the current user. If possible, return the git username, otherwise return the nci username.
+    """
+    dirname = os.path.dirname(file)
+
+    try:
+        name = (
+            subprocess.check_output(["git", "-C", dirname, "config", "user.name"])
+            .decode("ascii")
+            .strip()
+        )
+    except subprocess.CalledProcessError:
+        name = os.environ["USER"]
```

Comment:
How about this so we always get NCI username? NB: untested
```suggestion
    name = os.environ["USER"]
    try:
        name += 
            "(" +
            (
                subprocess.check_output(["git", "-C", dirname, "config", "user.name"])
                .decode("ascii")
                .strip()
            )
            + ")"
    except subprocess.CalledProcessError:
        pass
```

Reply:
Ok i've done this now. Back to you @aekiss